docs/user/01-manager.md

```diff
@@ -11,6 +11,12 @@ The Telemetry module includes Telemetry Manager, a Kubernetes [operator](https:/
 
 ![Manager](assets/manager-lifecycle.drawio.svg)
 
+### Self Monitor
+
+The Telemetry module contains a self monitor, based on [Prometheus](https://prometheus.io/), to collect and evaluate metrics from the managed gateways and agents. The manager retrieves the current pipeline healthiness from the self-monitor and enriches the status of the pipeline resources and the module status.
+
+![Manager](assets/manager-arch.drawio.svg)
```

Comment:
Consistency: In the diagram, either use 3rd person ("scrapes") or infinitive ("get"), but don't mix them.

Reply:
done, please double-check

Reply:
looks good! 

"Gets" doesn't need to be capitalized though